documentation/course/e2/README.md

```diff
@@ -1798,6 +1932,21 @@ TODO @tchojnacki: Dodać diagram pakietów, opis architektury.
 
 TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
+### API
+
+| **Rola**    | **Metoda** | **Endpoint**                   | **Wymagania** | **Opis**                                                                                                                |
+| ----------- | ---------- | ------------------------------ | ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `guest`     | `GET`      | `/ext/v1/offers`               | TODO          | Pobranie listy dostępnych ofert biletowych.                                                                             |
+| `guest`     | `GET`      | `/ext/v1/offers/:id`           | TODO          | Pobranie informacji o ofercie biletowej.                                                                                |
+| `passenger` | `GET`      | `/ext/v1/tickets`              | TODO          | Pobranie listy zakupionych biletów.                                                                                     |
+| `passenger` | `POST`     | `/ext/v1/tickets`              | TODO          | Zakup biletu.                                                                                                           |
+| `passenger` | `GET`      | `/ext/v1/tickets/:id`          | TODO          | Pobranie informacji o bilecie.                                                                                          |
+| `passenger` | `POST`     | `/ext/v1/tickets/:id/validate` | TODO          | Walidacja biletu.                                                                                                       |
+| `inspector` | `POST`     | `/ext/v1/tickets/:id/inspect`  | TODO          | Inspekcja biletu.                                                                                                       |
+| `admin`     | `POST`     | `/ext/v1/offers`               | TODO          | Utworzenie nowej oferty biletowej.                                                                                      |
+| `admin`     | `PUT`      | `/ext/v1/offers/:id`           | TODO          | Aktualizacja oferty biletowej, jeśli nie została zakupiona przez żadnego pasażera.                                      |
+| `admin`     | `DELETE`   | `/ext/v1/offers/:id`           | TODO          | Usunięcie oferty biletowej, jeśli nie została zakupiona przez żadnego pasażera, w przeciwnym razie jego zdezaktywowanie |
```

Comment:
I'm not sure an endpoint doing two things is a good idea

Reply:
How about instead of calling it a deletion, we said that offers *can only* be deactivated?

Reply:
I agree, let's just deactivate, the database storage we'd lose is negligible.